index.html

```diff
@@ -2472,24 +2479,32 @@ <h3>Data Schemas</h3>
     "degree": {
       "type": "ExampleBachelorDegree",
       "name": "Bachelor of Science and Arts"
+    },
+    "university": {
+      "type": "ExampleUniversity",
+      "name": "University of Examples, W3C"
```

Comment:
Move this "university" block inside of the "degree" credential, otherwise you're not tying the degree to the university (and vice-versa).

Reply:
I believe that was my intention--I want to show that there can be related data in the same credential, covered by two distinct JSON schemas.

The tying happens when the issuer signs over the entire subject IMO. 

For example, the first schema for the degree may be....
```json
{
    "$id": "https://example.com/schemas/degree-credential-schema.json",
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "name": "Degree VC Schema",
    "type": "object",
    "properties": {
      "credentialSubject": {
        "type": "object",
        "properties": {
          "degree": {
            "type": "object",
            "properties": {
              "type": { "type": "string"},
              "name": { "type": "string"}
            },
             "required": ["type", "name"]
          }
        },
        "required": ["degree"]
      }
    }
```

And the second, for the university...
```json
{
    "$id": "https://example.com/schemas/degree-credential-schema.json",
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "name": "University VC Schema",
    "type": "object",
    "properties": {
      "credentialSubject": {
        "type": "object",
        "properties": {
          "university": {
            "type": "object",
            "properties": {
              "type": { "type": "string"},
              "name": { "type": "string"}
            },
             "required": ["type", "name"]
          }
        },
        "required": ["university"]
      }
    }
```

However, if the university block was inside degree it would be awkward to have this represented by two schemas, though possible.

Reply:
I agree with @msporny ... however... this kinda highlights why just doing JSON-LD is not enough... 

You need to *think* about the RDF graph the JSON-LD becomes... in order to ensure that claims attribute to the correct `id`.... out spec says a bunch of informative stuff about this, which implementers ignore (because they are not required to understand the RDF graph)... but then we see a lot of comments like this... all over the place on implementations and profiles of the spec... it is a hint at a missing normative requirement.

You current example says:

`did:example:ebfeb1f712ebc6f1c276e12ec21` has a `degree`
`did:example:ebfeb1f712ebc6f1c276e12ec21` has a `university`

I guess if `did:example:ebfeb1f712ebc6f1c276e12ec21` is some identifier for a digital diploma this makes sense... so the example is valid... the comments on architecture of claims still hint at potentially providing better guidance, normatively... without some normative reason to follow them I think its safe to ignore them.

Reply:
<img width="1412" alt="Screen Shot 2023-07-12 at 5 08 29 PM" src="https://github.com/w3c/vc-data-model/assets/8295856/34b5d334-73a5-4154-af18-cfd66dfa484d">

Reply:
Yeah, after reflection, I better understand the point @msporny is making, but also agree with you @OR13 it seems like there needs to be normative guidance around this. Without it, I can see a credential being interpreted in multiple different ways.

As I have it written, it doesn't make much sense when viewed as an RDF graph. I'm wondering if there's another example I can use with separate top-level claims. Maybe both a degree and a name credential would be less controversial.

Reply:
Another version of the same confusion can be found here: https://github.com/w3c/vc-data-model/pull/1172#discussion_r1261786118

Reply:
I think the main difference is that the VCDM requires the connections between things to be explicit in how the modeling occurs in the data vs. just being in the head of the author. You don't have to guess what the author intended for how things "could be interpreted", but rather, it's in the data itself based on the required patterns. And, it's more of a consequence of that -- that we can build tools that will show the relationships between the data as a graph of information. You can't do that when the "interpretation" is in the author's head and not the structure of the data itself.